components/core/DownloadDaemon.py

```diff
@@ -1,16 +1,17 @@
 import threading
 import queue
-import json, inspect, os, time
+import json, time
 from DownloadManager import *
 from Models import Status, Download
 from EMail import send_mail
 from DiskMan import *
+from ConfReader import conf_reader
 
 import websocket
 import sys
 
 
-conf = {}
+conf = conf_reader("dl.conf")
```

Comment:
Since this method returns an object how about get_conf_reader or create_conf_reader?